frontend/src/index.tsx

```diff
@@ -14,6 +14,14 @@ import { Register } from "./pages/Register";
 import { SignIn } from "./pages/SignIn";
 import { Login } from "./pages/Login";
 import { ResetPassword } from "./pages/ResetPassword";
+// User account page
```

Comment:
Komentarz jest tutaj zbędny. Ścieżka + nazwa powinny być wystarczającym opisem samym w sobie. Jeżeli nie są to trzeba przemyśleć strukturę i nazwy